dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/ConnectorArguments.java

```diff
@@ -229,6 +232,37 @@ public class ConnectorArguments extends DefaultArguments {
           .withOptionalArg()
           .describedAs("sekr1t");
 
+  private final OptionSpec<ZonedDateTime> optionStartDate =
+      parser
+          .accepts(
+              OPT_START_DATE,
+              "Inclusive start date for data to export, value will be truncated to hour")
```

Comment:
This description is very generic and suggest that these parameters can be used to limit time range of data exported by the dumper but actually they are used only by the Airflow connector. I suggest to mention that they are valid for airflow connector only or support them in all other connectors (in addition to query-log-*).

Did you think about making the `query-log-*` flags deprecated? The one ones could eventually replace existing ones.